gstreamer/classify.py

```diff
@@ -35,6 +39,23 @@ def generate_svg(size, text_lines):
       dwg.add(dwg.text(line, insert=(10, y*20), fill='white', font_size='20'))
     return dwg.tostring()
 
+def output_tensor(interpreter):
```

Comment:
Move outout_tensor() to common.py and merge with the other output_tensor(interpreter, i)
(and invoke it here with i=0).